### ISSUE TITLE
dotcontent viewtool method of pullRelated is not honoring the languageId that is passed in.

### ISSUE THREAD

**Issue by brentgriffin** [dotCMS/core#19774]
**Describe the bug**
dotcontent viewtool method of pullRelated is not honoring the languageId that is passed in.

For example: Have related content that is both in English and Spanish.  When a languageId of "\*" is passed into the query, it only returns English content because the dotCMS code is manually inserting "languageId: 1" (the language from the session) into the condition query.  This results in the condition query containing both "languageId:*" and "languageId:1" which means only content with a languageId of 1 will be returned.  Inserting the session language into the query should only happen if "languageId" is not already included in the condition query.

Another example:  when a languageId other than the languageId of the session is passed in the condition query, no results are returned because the condition query ends up containing two different languageIds which matches exactly zero contentlets.

**To Reproduce**
1. Create Content Types:
- RegionalSpecialization
- IndexBucket
- On RegionalSpecialization type add a many to many relationship field called "indextest"
2. Create five instances of IndexBucket content (4 in English and 1 in Spanish)
3. Create instance of RegionalSpecialization content and relate to all 5 IndexBucket contentlets
4. Create page with this VTL (modifying the identifier to match your RegionalSpecialization identifier)
```
<h3> pullRelated with wildcard </h3>
Hard Coded Query: ("RegionalSpecialization.indextest","0fcf4a2c-18a1-484f-8749-d586ffcfff95", "+languageId:*",false,0,"")
<br>
<br>
#set($indicesList = $dotcontent.pullRelated("RegionalSpecialization.indextest","0fcf4a2c-18a1-484f-8749-d586ffcfff95", "+languageId:*",false,0,""))   
Indices List Size: $indicesList.size()
```
5. Request page with VTL in the browser of your choice
6. Notice that you are only receiving 4 contentlets back instead of 5 - only the english (default language) contentlets are returned.

**Expected behavior**
When a languageId is passed into an ES content pull, expect for it to be honored.

**Recommended Changes**
In [ContentTool](https://github.com/dotCMS/core/blob/v5.3.4.1/dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/ContentTool.java#L383-L388) recommend adding logic similar to this:
```
long langId = language.getId();
if(condition.contains("languageId")) {
    langId =-1;
}
// pass langId rather than language.getId() to pullRelated.
```

In [ContentUtils](https://github.com/dotCMS/core/blob/v5.3.4.1/dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/util/ContentUtils.java#L560-L562) modify it to look something like this:
```
                if (language != -1 && !condition.contains("languageId")){
                    pullQuery.append(" ").append("+languageId:").append(language).append(" ");
                }
```

**Comment by bryanboza** [comment]
Fixed, unable to reproduce after the last changes. Now we are including the pieces of content in other languages

**Comment by erickgonzalez** [comment]
Fixed in 21.04 only needed in 5.3.8 LTS
